src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const values = document.querySelectorAll('.population');
+const avarage = document.querySelector('.average-population');
```

Comment:
The variable name 'avarage' is misspelled. It should be 'average' to match the intended purpose and improve code readability.